example/src/pages/Modules.tsx

```diff
@@ -100,6 +101,25 @@ const renderModulePaymentNotice = () => (
   </>
 );
 
+const renderModuleCheckout = () => (
+  <>
+    <ComponentViewerBox name="ModuleCheckout, default">
+      <ModuleCheckout
+        paymentLogo="amex"
+        header="Amex"
+        body="arien_c********@**hoo.it"
```

Comment:
For consistency, we should use the same prop names used in `ListItemTransaction` or `ListItemNav`, especially if they are conceptually similar:
```suggestion
        title="Amex"
        subtitle="arien_c********@**hoo.it"
```
or
```suggestion
        value="Amex"
        description="arien_c********@**hoo.it"
```